tests/cover/test_numerics.py

```diff
@@ -19,9 +19,12 @@
 
 import math
 
+import pytest
+
 from hypothesis import given, assume
-from tests.common.utils import fails, fails_with
-from hypothesis.strategies import decimals, fractions, float_to_decimal
+from tests.common.utils import fails
+from hypothesis.strategies import decimals, fractions
+from hypothesis.internal.compat import float_to_decimal
 
 
```

Comment:
A thing that I *think* is fine but would like to see some explicit tests for is making sure that things work correctly when the number of possible values is very small - e.g. what happens when you have `decimals(min_value=0.1, max_values=0.2, places=1)` (so the only valid values are `[0.1, 0.2]`)?